.travis.yml

```diff
@@ -1,13 +1,31 @@
 language: perl
+# Archive::Zip currently requires sudo
+sudo: required
+os:
+  - linux
 perl:
+  - "blead"
+  - "5.24"
+  - "5.22"
+  - "5.20"
   - "5.18"
   - "5.16"
   - "5.14"
   - "5.12"
   - "5.10"
+matrix:
+   allow_failures:
+      - perl: 'blead'
```

Comment:
Its ok if blead perl fails - dont consider that single fail to mean the whole build testing has failed for this project